packages/clerk-js/src/ui/components/SignUp/SignUpRestrictedAccess.tsx

```diff
@@ -18,22 +25,36 @@ export const SignUpRestrictedAccess = () => {
             icon={Block}
             sx={t => ({
               margin: 'auto',
-              width: t.sizes.$12,
-              height: t.sizes.$12,
+              width: t.sizes.$10,
+              height: t.sizes.$10,
```

Comment:
Does this mean this icon, has a different size from (let's say) the SignIn component ?

Reply:
Nvm, i thought it was the logo